src/Components/JobsContainer/Internal/JobsList/JobsList.types.ts

```diff
@@ -1,11 +1,11 @@
 import { IStyle, IStyleFunctionOrObject } from '@fluentui/react';
-import { IAdapterData, IJob, IUseAdapter } from '../../../../Models/Constants';
+import { IAdtApiJob } from '../../../../Models/Constants';
 import { IExtendedTheme } from '../../../../Theming/Theme.types';
 
 export interface IJobsListProps {
-    jobs: IJob[];
-    onCancelJob: IUseAdapter<IAdapterData>;
-    onDeleteJob: IUseAdapter<IAdapterData>;
+    jobs: IAdtApiJob[];
```

Comment:
same comment here, should use the local client type, not the API type

Reply:
same as above